testpr-script.js

```diff
@@ -0,0 +1,63 @@
+const { exec, spawn } = require('child_process');
+const { argv } = require('yargs');
+const username = require('username');
+
+const id = argv._[0];
+
+function spawnPromise(command, args) {
+  return new Promise((resolve, reject) => {
+    const p = spawn(command, args);
+
+    p.stdout.on('data', data => {
+      console.log(data.toString());
+    });
+
+    p.stderr.on('data', data => {
+      console.error(data.toString());
+    });
+
+    p.on('exit', code => {
+      if (code === 0) {
+        resolve();
+      } else {
+        reject();
+      }
+    });
+  });
+}
+
+async function test(prId) {
+  const branchName = `pr-${username.sync()}-${prId}`;
+
+  exec(
+    `git fetch origin pull/${prId}/head:${branchName}`,
```

Comment:
@christianalfoni This wont work on forks I guess? 🤔

Reply:
It is for forks :) Take a look at: https://github.com/codesandbox/codesandbox-client/pull/2787

Reply:
I mean if I run locally `yarn test:pr 2787` this wont work, since my `origin` is my own fork of the repo and my `upstream` is the original repo

Reply:
Oh, I see :) Yeah, the intention was that the devs who merge into `master` has a chance to verify the PR before they do so :)

Reply:
If it's only for CodeSandbox-people, my comment is obsolete 😉

Reply:
I did not want to use the term `Codesandbox-people`, hehe... "merge people" 😂

Reply:
Same people, different name 😉